### PR TITLE
Corrected how we submit the oracle.can_connect check

### PR DESCRIPTION
**Pull request by aldrickdev**
### What does this PR do?
This PR will correct how we submit the oracle.can_connect so that the expected `host` tag is added.

### Motivation
[SDBM-1373](https://datadoghq.atlassian.net/browse/SDBM-1373)

### Describe how you validated your changes
The changes can be validated by making sure that the value for the `host` tag on the `oracle.can_connect` has the value of the database instance.

### Additional Notes

[SDBM-1373]: https://datadoghq.atlassian.net/browse/SDBM-1373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lu-zhengda — APPROVED** [review]

**Comment by nenadnoveljic** [comment]
Can you add a test case?

File: pkg/collector/corechecks/oracle/custom_queries_test.go [hunk 1]
```diff
@@ -57,6 +57,15 @@ func TestCustomQueries(t *testing.T) {
 	sender.AssertMetricTaggedWith(t, "Gauge", "oracle.custom_query.test.c1", []string{"c2:A"})
 }
 
+func TestCanConnectTags(t *testing.T) {
```
Comment (amw-zero) [inline pkg/collector/corechecks/oracle/custom_queries_test.go:60]: Let's move this into a more appropriate file (it's currently with custom query tests)

File: pkg/collector/corechecks/oracle/oracle_integration_test.go [hunk 2]
```diff
@@ -424,6 +424,15 @@ func buildConnectionString(connectionConfig config.ConnectionConfig) string {
 	return connStr
 }
 
+func TestCanConnectTags(t *testing.T) {
+	chk, sender := newDefaultCheck(t, "", "")
+	err := chk.Run()
+	require.NoError(t, err)
+
+	expectedHostTag := fmt.Sprintf("host:%s", chk.config.InstanceConfig.Server)
+	sender.AssertServiceCheck(t, "oracle.can_connect", servicecheck.ServiceCheckOK, "", []string{expectedHostTag}, "")
```
Comment (nenadnoveljic) [inline pkg/collector/corechecks/oracle/oracle_integration_test.go:433]: I think we should assert the host name instead of `""`

**Review by nenadnoveljic — APPROVED** [review]

**Comment by nenadnoveljic** [comment]
Some labels and release note are missing.

**Review by michaelcretzman — APPROVED** [review]

**Comment by nenadnoveljic** [comment]
@aldrickdev Thanks for fixing this! Ready to merge.

**Pull request merged**